data_sources/kafka_data_source.py

```diff
@@ -0,0 +1,61 @@
+from data_sources.data_source import DataSource
+from confluent_kafka import Consumer, KafkaError
+import time
+import json
+
+
+class KafkaDataSource(DataSource):
+    def __init__(self, kafka_endpoint, kafka_topic, s3_bucket, read_timeout_secs, batch_size = 1024):
```

Comment:
1. Is it common to read only from one topic or there are cases when you'll read from more than one?
2. also here you mentioned s3 specifically, should it be generic?
3. Shouldn't read_timout_secs also have default?

Reply:
1. usually you read from one topic
2. same as bafore
3. I will add